build/Signing.cs

```diff
@@ -0,0 +1,185 @@
+using System;
```

Comment:
I find the formatting of this file to be very odd - I'm not going to say change it, but I'd consider the indenting here to be too excessive.

Reply:
Sorry about that - I've been fighting Rider to give me good auto formatting. I think it looks better now.